### ISSUE TITLE
Too slow :(

### ISSUE THREAD

**Issue by mattfaraday** [ZitaoTech/Hackberry-Pi_Zero#8]
The device is awesome. But it's just too slow. You can't run browsers etc on it as it even warns you about the lack of RAM for the job. 
Have you tried any other single board computers the same form factor? like Radxa Zero 3 W

**Comment by ZitaoTech** [comment]
That's right, the biggest issue is rpi02w has only 512MB ram, it would be much better if it has 1GB ram. Also, are you running on 64bit OS? I found 32bit OS should be faster on rpi2w and you can try dietpi OS, which is more lightweighted. And for browser I would recommend web browser in terminal like w3m. 
Radxa zero3w doesn't support the dpi interface. It's difficult to drive the screen

**Comment by huuck** [comment]
Does dietPi come in a 32bit flavour? Could not find it anywhere on their website.

**Comment by markuman** [comment]
I guess CM4 version would solve the issue.
